.github/workflows/deploy-pangeo-181919.yaml

```
@@ -0,0 +1,30 @@
+name: Deploy and Test pangeo hub
```

```suggestion
name: Deploy and Test COESSING hub
```

Just so we don't get confused with naming!